crowbar.yml

```diff
@@ -26,6 +26,6 @@ barclamp:
 crowbar:
   layout: 1
   order: 99
-  run_order: 99
-  chef_order: 99
+  run_order: 2000
+  chef_order: 2000
```

Comment:
I guess you're doing that to disable the services and not have them restarted by later recipes. That won't work for anything that happens in notifications, though.

The right "fix" here is that all the other (at least, the openstack ones) cookbooks should stop being applied. Not sure how to best do this.